CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ### v4.0.0 Sep 1st, 2022
 
 * Update package dependencies to latest versions for Null Safety support.
-* Update application UI 
+* Update example folder UI 
```

Comment:
remove "folder", we are updating the UI of the example, not the folder.

Reply:
done